kommunicate/src/main/java/io/kommunicate/KmConversationHelper.java

```diff
@@ -263,7 +268,7 @@ private static KMStartChatHandler getStartChatHandler(final boolean isSkipChatLi
             public void onSuccess(Channel channel, Context context) {
                 try {
                     if (resultReceiver != null) {
-                        resultReceiver.send(LeadCollectionActivity.PRECHAT_RESULT_CODE, null);
+                        resultReceiver.send(KmConstants.PRECHAT_RESULT_CODE, null);
```

Comment:
It is considered a bad practice to keep all constants in a single file, constants should be placed where they belong to avoid any dependencies between different module or packages of code. https://github.com/Kommunicate-io/Kommunicate-Android-Chat-SDK/pull/59#pullrequestreview-278956711

Reply:
I guess its OK to keep constants shared across different classes in a single file. Since these constants are being referred to in a dependent module, I have placed them in a single file.

Reply:
Earlier these constants were defined in their respective classes. But now the dependencies have been reversed and thus those constants cannot be accessed from the classes that are in kommunicate module. If we move these to original classes then we will have to declare these constants again in the class that is accessing them which is similar to declaring them in a single file.